app/controllers/EventsController.scala

```diff
@@ -31,19 +35,21 @@ import scala.concurrent.ExecutionContext.Implicits.global
 
 object EventsController extends EventsController {
   val eventsService: EventsService = EventsService
-  val locationsAndVenues: LocationsWithVenuesRepository = LocationsWithVenuesInMemoryRepository
+  val locationsAndVenuesRepository: LocationsWithVenuesRepository = LocationsWithVenuesInMemoryRepository
+  val assessorAllocationService: AssessorAllocationService = AssessorAllocationService
 }
 
 trait EventsController extends BaseController {
   def eventsService: EventsService
-  def locationsAndVenues: LocationsWithVenuesRepository
+  def locationsAndVenuesRepository: LocationsWithVenuesRepository
+  def assessorAllocationService: AssessorAllocationService
 
   def venuesForEvents: Action[AnyContent] = Action.async { implicit request =>
```

Comment:
I can see locations and venues being used by other things and hence should probably be exposed through a custom endpoint/controller. I think it is fine to leave it here for now until we know for sure.